guides/supporters.rst

```diff
@@ -52,6 +52,7 @@ You guys are awesome!
 - DrZzs
 - fabi
 - Franck Nijhof
+- Gaiar Baimuratov
```

Comment:
It looks like you created this PR from the `current` branch. Please rebase this PR on the `next` branch (see contributing docs)

Reply:
This still applies